### PR TITLE
Update python.d README 

### PR DESCRIPTION
**Pull request by OdysLam**
##### Summary
Underline in README the existence of multiple url plugins to facilitate plugin developers and offered 2 examples.

##### Component Name
docs
##### Additional Information

Added also a comment regarding the existence of custom variables in the config. 

Thanks @ilyam8 for the info

### CODE REVIEW

File: collectors/python.d.plugin/README.md [hunk 1]
```diff
@@ -197,10 +197,14 @@ For additional security it uses python `subprocess.Popen` (without `shell=True`
 
 _Examples: `apache`, `nginx`, `tomcat`_
 
+_Multiple Endpoints (urls) Examples: `rabbitmq` (simpler) , `elasticsearch` (threaded)_
```
Comment (ilyam8) [inline collectors/python.d.plugin/README.md]: I think it would be better to make them hyperlinks.

**Comment by ilyam8** [comment]
@OdysLam CLA is not signed

**Comment by OdysLam** [comment]
I used a different mail to commit and the CLA won't recognise it even after I added it to github and configured it as my main mail, weird.

Will try again later.

**Comment by ilyam8** [comment]
it is signed

**Comment by joelhans** [comment]
I see two errors in the Netlify build:

```
6:05:27 AM:    - ERROR: ./collectors/python.d.plugin/README.md - https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/rabbitmq is an absolute link to a Netdata file. Please convert to relative.
6:05:27 AM:    - ERROR: ./collectors/python.d.plugin/README.md - https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/elasticsearch is an absolute link to a Netdata file. Please convert to relative.
```

Think you could fix these? You can most likely change them to just `rabbitmq/` and `elasticsearch/`, respectively. Maybe `rabbitmq/README.md` and `elasticsearch/README.md`.

**Comment by OdysLam** [comment]
I am sorry for the commit spam, but it was the first time that I worked with relative paths in github doc and I couldn't test it locally before committing.

**Comment by joelhans** [comment]
No worries! I can be confusing even for me, and I can build it locally. I actually think the correct syntax for that entire line would be:

```
_Multiple Endpoints (urls) Examples: [`rabbitmq`](rabbitmq/README.md) (simpler) , [`elasticsearch`](elasticsearch/README.md) (threaded)_
```

The links just need to be relative from the `.md` file you're editing—no need to start at the root.

If that doesn't work, I'll download your fork and test it locally to figure it out. Or, if you're on Linux and are a brave soul, you can try [building the docs locally](https://docs.netdata.cloud/docs/contributing/contributing-documentation/#build-the-documentation) yourself.

**Comment by OdysLam** [comment]
Indeed you are correct.

It is worth mentioning that although I setup my system to locally build the docs, I observed that the markdown editor I use (macdown) correctly linked the files, thus when I clicked on the linkes in the preview file it took me to the local directory where the docs are found. Very interesting!

Sorry for all this hassle.

Best,
Odysseas

**Comment by joelhans** [comment]
Netlify is still failing on the deploy, but not because of the work in this PR. We have some issues with our localization repo and this will merge/build just fine.

**Pull request merged**